Common/StringUtils.cpp

```diff
@@ -316,12 +316,17 @@ void SplitString(std::string_view str, const char delim, std::vector<std::string
 
 void SplitString(std::string_view str, const char delim, std::vector<std::string> &output) {
 	size_t next = 0;
-	for (size_t pos = 0, len = str.length(); pos < len; ++pos) {
-		if (str[pos] == delim) {
-			output.emplace_back(str.substr(next, pos - next));
-			// Skip the delimiter itself.
-			next = pos + 1;
+	size_t pos = 0;
+	size_t len = str.length();
+	output.reserve(len);
```

Comment:
that's a really large reservation, as if every character would be split into their own string...